integration/channelparticipation/channel_participation.go

```diff
@@ -158,3 +162,26 @@ func channelInfoShortMatcher(channel string) types.GomegaMatcher {
 		"URL":  Equal(fmt.Sprintf("/participation/v1/channels/%s", channel)),
 	})
 }
+
+// JoinOrdererJoinPeersAppChannel Joins an orderer to a channel for which the genesis block was created by the network
+// bootstrap. It assumes a channel with one orderer. It waits for a leader (single orderer, always node=1), and then
+// joins all the peers to the channel.
+func JoinOrdererJoinPeersAppChannel(network *nwo.Network, channelID string, orderer *nwo.Orderer, ordererRunner *ginkgomon.Runner) {
```

Comment:
This is a general thought, and less tied to this specific function, but - given we're removing the system channel, is there a point in having joining utility functions such as this, in the channel participation package of integration? Channel participation is all that is left, so shouldn't we extend nwo.Network to have these utilities instead, or at least have joining capabilities in a different package, as this is now essential and going to be widely used everywhere in the tests?

Reply:
Absolutely correct. 

The end game is to have all this in `nwo` and replace the `nwo.Create/Join Channel` etc. 
However, right now the integration/channelparticipation depends on nwo, I suppose whoever wrote  it in the first place had a cyclic dependency issue. So my plan is to extend this while  we still have tests that use the system channel, and once we finish that clean up, do some refactoring and tidy up the package structure so that all those utility functions are in `nwo`.